scripts/takepeds

```diff
@@ -31,8 +31,8 @@ HUTCH=${EXP:0:3}
 echo $PYCMD -i "${HUTCH^^}" -u `whoami` -e "$EXP"  -t DARK  -r $RUN -m "$elogMessage"
 $PYCMD -i "${HUTCH^^}" -u `whoami` -p pcds -e "$EXP"  -t DARK  -r $RUN -m "$elogMessage"&
 
-echo 'please call: makepeds -r '`get_lastRun`' -u <userID>'
-echo 'for gain-switching detectors we recommend: makepeds -u <userID> -q milano -r '`get_lastRun`
+echo 'Please call: makepeds -q milano -r '`get_lastRun`' -u <userID>'
+echo 'To run job in the interactive node run (slower): makepeds -r '`get_lastRun`' -u <userID>'
```

Comment:
Actually, for non-gainswitching detectors it's really not clear to me if interactive is faster or slower than the queue as this'll depend on the other uses of the node where the job runs & the SLURM overhead. Maybe just don't say anything and assume that people know that not passing a queue will make the job run interactively (which we'll write on the pedestal page)?

Reply:
I agree, let's take that first option out altogether!